src/librustc_mir/transform/rustc_peek.rs

```diff
@@ -86,9 +67,12 @@ impl<'tcx> MirPass<'tcx> for SanityCheck {
         if has_rustc_mir_with(&attributes, sym::rustc_peek_definite_init).is_some() {
             sanity_check_via_rustc_peek(tcx, body, def_id, &attributes, &flow_def_inits);
         }
+        // FIXME: Uncomment these as analyses are migrated to the new framework
+        /*
         if has_rustc_mir_with(&attributes, sym::rustc_peek_indirectly_mutable).is_some() {
```

Comment:
so wait, not all of the rustc peek tests are  completely  ported over then, right?

Reply:
Correct. `IndirectlyMutableLocals` remains, and will be the next PR I submit.

Reply:
See #69113.